src/viam/sdk/common/proto_type.hpp

```diff
@@ -28,6 +29,16 @@ class ProtoType {
     google::protobuf::Value proto_value();
     friend bool operator==(const ProtoType& lhs, const ProtoType& rhs);
 
+    template <typename T>
```

Comment:
This could probably be deferred into the tflite review. But as it happens, there is no way to actually get values out of a `ProtoType` object. This provides the nullptr-on-type-mismatch API.